### PR TITLE
Merge from features/authenticate to develop

### PR DESCRIPTION
**Pull request by Luwat**
Kindly help me review this branch before merging.

### CODE REVIEW

File: app/(auth)/sign-in.tsx [hunk 1]
```diff
@@ -5,26 +5,78 @@ import { useState } from "react";
...
+        return;
+      }
+
+      if (form.password.length < 8) {
+        setError("Password must be at least 8 characters long");
+        return;
+      }
+
+      if (!/^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(form.email)) {
+        setError("Invalid email address");
```
Comment (michael-rivala) [inline app/(auth)/sign-in.tsx:35]: Nice work done here!!
Check if there are packages to handle input validation though to make your work easier

Comment (michael-rivala) [reply app/(auth)/sign-in.tsx:35]: You don't have to do that on this project. Just check it out

**Review by michael-rivala — APPROVED** [review]
Well done!!
I have approved this but kindly check the comment.

File: app/(auth)/sign-in.tsx (hunk 1 above)
Comment (Luwat) [reply app/(auth)/sign-in.tsx:35]: Well noted. Thanks

**Pull request merged**